Formula/yosys.rb

```diff
@@ -0,0 +1,77 @@
+class Yosys < Formula
+  desc "Framework for Verilog RTL synthesis"
+  homepage "http://www.clifford.at/yosys/"
+  url "https://github.com/cliffordwolf/yosys/archive/yosys-0.7.tar.gz"
+  sha256 "3df986d0c6bf20b78193456e11c660f2ad935cc126537c2dc5726e78896d6e6e"
+
+  option "without-abc", "Build without support for ABC"
+  option "without-tcl", "Build without support for Tcl language bindings"
```

Comment:
I included the options because the Makefile supports them and because the :recommended dependencies can be disabled. There's no great need that I can think of. I'll take them out.